lib/folio_record.rb

```diff
@@ -87,6 +87,19 @@ def sirsi_holdings
       _current_library, current_location = LocationsMap.for(item.dig('location', 'temporaryLocation', 'code'))
       current_location ||= Folio::StatusCurrentLocation.new(item).current_location
 
+      # TODO: we don't handle multiple courses for a single item, because it's beyond parity
+      item_courses = courses.select { |course| course[:listing_id] == item['courseListingId'] }
+      course = item_courses.first
```

Comment:
```suggestion
      course = courses.first { |course| course[:listing_id] == item['courseListingId'] }
```

Reply:
Did not know you could pass that a block! Cool!